### DISCUSSION TITLE
Example using Hugging Face Large Language Models?

### DISCUSSION THREAD

**Discussion by mrdunky** [xamarin/TorchSharp#1013] [General]
Hi, 
I recently saw a great video showing how to use the Hugging Face Large Language models from PyTorch. 
They used very simple python code like this below. 

Is this possible to do with TorchSharp? 
I would really love to have a go with this it if is possible. 
Thank you!

`from transformers import AutoTokenizer, AutoModelForSeq2SeqLM, GenerationConfig
line = 'What color is the undoubtedly beautiful sky?'
model_name = 'google/flan-t5-xl'
tokenizer = AutoTokenizer.from_pretrained(model_name)
model = AutoModelForSeq2SeqLM.from_pretrained(model_name)
config = GenerationConfig(max_new_tokens=200)
...`
This was from this excellent video [https://www.youtube.com/watch?v=tL1zltXuHO8](url)

**Comment by GeorgeS2019** [comment]
https://github.com/dotnet/TorchSharpExamples/pull/19
https://github.com/dotnet/TorchSharp/discussions/967
https://github.com/dotnet/TorchSharp/discussions/457

**Reply by mrdunky** [reply]
Thank you so much. I should have searched better. 
Appreciate you taking the time!

**Reply by GeorgeS2019** [reply]
@mrdunky  do not close the discussion, other experts will help you

**Comment by mrdunky** [comment]
oh, thank you!
